src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -1052,7 +1061,7 @@ private void onHandleChainInventoryMessage(PeerConnection peer, ChainInventoryMe
     try {
       if (peer.getSyncChainRequested() != null) {
         Deque<BlockId> blockIdWeGet = new LinkedList<>(msg.getBlockIds());
-        if (blockIdWeGet.size() > 0){
+        if (blockIdWeGet.size() > 0) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use isEmpty() to check whether the collection is empty or not. ([squid:S1155](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1155))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-0plHK86Hco_1tm&open=AWalU-0plHK86Hco_1tm)

[ISSUE_KEY]: AWalU-0plHK86Hco_1tm